torch_glow/src/FuseConcat.cpp

```diff
@@ -0,0 +1,52 @@
+#include "FuseConcat.h"
+namespace glow {
+bool isFusableConcatNode(torch::jit::Node *node) {
+  if (node->kind() != at::aten::cat ||
+      !node->is_constant(torch::jit::attr::dim)) {
+    return false;
+  }
+
+  auto input_node = node->namedInput(torch::jit::attr::tensors)->node();
+  if (input_node->kind() != at::prim::ListConstruct) {
+    return false;
+  }
+  if (input_node->output()->uses().size() > 1) {
+    return false;
+  }
+
+  return true;
+}
+
+torch::jit::Node *createFusedConcat(torch::jit::Node *node) {
+  AT_ASSERT(node->kind() == at::aten::cat);
+  torch::jit::Graph *graph = node->owningGraph();
+  torch::jit::Node *input_node =
+      node->namedInput(torch::jit::attr::tensors)->node();
+  int64_t dim = node->get<int64_t>(torch::jit::attr::dim).value();
+
+  torch::jit::Node *fused_cat =
+      graph
+          ->create(at::Symbol::fromQualString("glow::fused_concat"),
```

Comment:
How is this different from `prim::FusedConcat`

Reply:
Not quite different, prim::FusedConcat also works. But dont want to rely on this existing symbol, since it might have some other use in other scenario in pytorch, and might change in the future.

Reply:
Let's just use prim::FusedConcat if it does exactly what we want